powershell/plan.sh

```diff
@@ -0,0 +1,116 @@
+pkg_name=powershell
+pkg_origin=core
+pkg_version=6.0.0-alpha.12
+pkg_license=('MIT')
+pkg_upstream_url=https://msdn.microsoft.com/powershell
+pkg_description="PowerShell is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized for dealing with structured data (e.g. JSON, CSV, XML, etc.), REST APIs, and object models. It includes a command-line shell, an associated scripting language and a framework for processing cmdlets."
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source="https://github.com/PowerShell/PowerShell"
+pkg_deps=(
+  core/dotnet-core
+  core/gcc
+  core/glibc
+  core/gcc-libs
+  core/icu/52.1
+  core/util-linux
+  core/krb5
+  core/libunwind
+  core/lttng-ust
+  core/openssl
+)
+pkg_build_deps=(
+  core/patchelf
+  core/cmake
+  core/make
+  core/git
+)
+pkg_bin_dirs=(.)
+
+do_download() {
+  pushd $HAB_CACHE_SRC_PATH
+  rm -rf $pkg_dirname
+  mkdir $pkg_dirname
+
+  # Important to recursively clone submodules.
+  # This is why we do not download source
+  git clone -b v$pkg_version --recursive "$pkg_source"
+  popd
+  return 0
```

Comment:
You don't need to `return 0` here. We put those in when doing empty functions because bash does not allow empty function bodies. The function will return the return code of the last command in it.